src/web/mjs/connectors/Webmangatr.mjs

```diff
@@ -0,0 +1,64 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class webmangatr extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'webmangatr';
+        super.label = 'WEBMANGATR';
+        this.tags = ['Turkish' ,'webtoon','hentai'];
+        this.url = 'https://webmangatr.com';
+    }
+
+    async _getMangas() {
+        let mangaList = [];
+        let request = new Request(this.url + '/dizi-arsivi/', this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.col-lg-8 > a:last-of-type');
+        let pageCount = data[0].getAttribute('data-max');
+        for(let page = 1; page <= pageCount; page++) {
+            let mangas = await this._getMangasFromPage(page);
+            mangaList.push(...mangas);
+        }
+        return mangaList;
+    }
+
+    async _getMangasFromPage(page) {
+        let request = new Request(`${this.url}/dizi-arsivi/page/${page}/`, this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.movie-details > div.name > a');
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
+                title: element.text.trim()
+            };
+        });
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'h1.film');
+        let id = uri.pathname + uri.search;
+        let title = data[0].textContent.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getPages(chapter) {
+        let script = `
+            new Promise(resolve => resolve( [...document.querySelectorAll('p > img[loading]')].map(image => image.dataset.src)))
+        `;
+        let request = new Request(this.url + chapter.id, this.requestOptions);
+        let data = await Engine.Request.fetchUI(request, script);
+        return data.map(page => this.createConnectorURI(page));
```

Comment:
It creates a connector URL, but no handler is implemented in this class, so it will use the base handler.
Is this intentional?